driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTestModifications.java

```diff
@@ -168,8 +168,8 @@ public static void applyCustomizations(final TestDef def) {
                 .when(() -> def.isReactive() && UnifiedTest.Language.KOTLIN.equals(def.getLanguage()))
                 .file("crud", "findOne");
 
-        def.skipJira("https://jira.mongodb.org/browse/JAVA-5827")
-                        .file("crud", "bypassDocumentValidation");
+        def.skipNoncompliant("Find with batchSize equal to limit - batchSize is set to limit if limit is smaller")
+                .test("crud", "find", "Find with batchSize equal to limit");
```

Comment:
I'm not sure I understand the rationale - in [the test](https://github.com/mongodb/specifications/blob/335848d3f4577724b7b82cb9fa0c12defba54647/source/crud/tests/unified/find.json#L292), the limit is not smaller, but is equal to the batchSize. What is the actual expected behaviour in the spec, is it to set batchSize to 1 over the limit, or something else? And why is the driver not doing that?

Reply:
Updated - I'm not sure why that was there, must have been an unneeded artifact left over when developing the initial submodule code.